client/lib/track-element-size/index.ts

```diff
@@ -76,7 +76,7 @@ export function useWindowResizeCallback(
 			window.removeEventListener( 'resize', throttledMeasureElement );
 			throttledMeasureElement.cancel();
 		};
-	}, [ elementRef.current, callback ] );
+	}, [ callback ] );
```

Comment:
eslint@6 "fixed" this, but it seems to be wrong? at least it broke the test...

Reply:
Right, I don't think that's a valid fix. It seems you dropped it for now, though?

Reply:
yup, dropped for now. curious what it'll do with the v2 of the hooks plugin.

Reply:
@sgomes would you mind trying https://github.com/Automattic/wp-calypso/pull/37404 and see if that changes how it "fixes" things?

Reply:
As far as I can tell, there's no fix anymore (assuming my IDE is picking it up correctly). The code I wrote is probably incorrect, though; I'll need to take a closer look at this.